modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -87,3 +87,11 @@ To create your app, follow these steps:
 . Click *Create App*.
 . Create your app by completing the fields in the *Create App* page.
 . Click *Save*.
+
+== Application Whitelisting
```

Comment:
Whitelist or Whitelisting? Let's keep it consistent, but either one is fine from a docs perspective.

Reply:
I figured whitelisting is the action - the whitelist is the object that gets modified when the admin goes out and does some whitelisting

Reply:
```suggestion
== Application Whitelist
```